package.json

```diff
@@ -13,19 +13,15 @@
     "dev:web": "npm-run-all --parallel dev:web:*",
     "dev:node": "node ./src/node/command.mjs ",
     "dev": "npm run dev:web",
-    "build:php:web": "cd wasm-build/php && bash webworker-build-wasm.sh",
-    "build:php:node": "cd wasm-build/php && bash node-build-wasm.sh",
+    "build:php:web": "cd wasm-build/php && bash webworker-build-wasm.sh && bash web-publish.sh",
+    "build:php:node": "cd wasm-build/php && bash node-build-wasm.sh && bash node-publish.sh",
```

Comment:
Since the npm script is called `build`, let's call the files consistently: `web-build.sh` and `node-build.sh`.

Reply:
After some more thought I like your idea better – these scripts do publish the files after all.